package.json

```diff
@@ -64,6 +64,6 @@
   "peerDependencies": {
     "vue": "^2.5.0",
     "vuex": "^3.0.0",
-    "vue-class-component": "^6.0.0"
+    "vue-class-component": "^7.0.0"
```

Comment:
Could you include `^6.0.0` because it should be compatible with both? So it will be `^6.0.0 || ^7.0.0`

Reply:
done!